packages/shared/src/features/onboarding/shared/StepHeadline.tsx

```diff
@@ -21,21 +21,23 @@ export type StepHeadlineProps = {
   description?: string;
   descriptionProps?: TypographyProps<TypographyTag.P>;
   heading: string;
+  className?: string;
```

Comment:
Extend type WithClassNameProp (or something along those lines) instead of adding manually

Reply:
Makes sense, yeah. I'll modify it.